bitnami/common/Chart.yaml

```diff
@@ -20,4 +20,4 @@ sources:
   - https://github.com/bitnami/charts
   - http://www.bitnami.com/
 type: library
-version: 1.3.7
+version: 1.3.8
```

Comment:
Could you please update the `appVersion` as well? Both `version` and `appVersion` should match.

Reply:
Done 👍